### PR TITLE
[AF-384] Scope localStorage per installation

### PR DESCRIPTION
**Pull request by danielbreves**
:v:

/cc @zendesk/vegemite

### References
* JIRA: https://zendesk.atlassian.net/browse/AF-384

### Risks
* [low] local storage may not work correctly for migrated v1 apps

### CODE REVIEW

**Comment by adammw** [comment]
Changelog?

**Comment by danielbreves** [comment]
@adammw did you just volunteer to create one? Nice!

**Comment by adammw** [comment]
Otherwise LGTM!

File: src/javascripts/base_app.js [hunk 1]
```diff
@@ -112,14 +112,16 @@ BaseApp.prototype = {
   },
 
   store: function(keyOrObject, value) {
+    var installationId = this._metadata.installationId;
     if (typeof keyOrObject === 'string') {
+      var key = `${installationId}:${keyOrObject}`;
```
Comment (sandlerr) [inline src/javascripts/base_app.js]: won't this be `[object Object]` now?

**Comment by sandlerr** [comment]
:+1: but if you change `var key` to `let key` then you won't be shadowing that variable when iterating over the object :D

**Comment by svizzari** [comment]
👍 though I agree with @sandlerr re: using `let`

**Comment by adammw** [comment]
👍

**Pull request merged**